### ISSUE TITLE
Module not found: Error: Can't resolve 'Vue' 

### ISSUE THREAD

**Issue by mazurva** [ankurk91/vue-loading-overlay#3]
```
ERROR in ./~/vue-loading-overlay/dist/vue-loading.min.js
Module not found: Error: Can't resolve 'Vue' in ./node_modules/vue-loading-overlay/dist'
 @ ./~/vue-loading-overlay/dist/vue-loading.min.js 1:82-96
 @ ./vue-source/src/main.js
```

**Comment by ankurk91** [comment]
Will look into that.
Are your using this package with a module bundler?

**Comment by mazurva** [comment]
My webpack.config.js
```
var path = require('path')
var webpack = require('webpack')

module.exports = {
    entry: {
        //client: './vue-source/client/main.js',
        //admin: './vue-source/admin/main.js',
        build: './vue-source/src/main.js',
    },
    output: {
        path: path.resolve(__dirname, './public/dist'),
        publicPath: '/dist/',
        filename: '[name].js'
    },
    module: {
        rules: [
            {
                test: /\.vue$/,
                loader: 'vue-loader',
                options: {
                    loaders: {}
                    // other vue-loader options go here
                }
            },
            {
                test: /\.js$/,
                loader: 'babel-loader',
                exclude: /node_modules/
            },
            {
                test: /\.(png|jpg|gif|svg)$/,
                loader: 'file-loader',
                options: {
                    name: '[name].[ext]?[hash]'
                }
            },
            {
                test: /\.css$/,
                use: [ 'style-loader', 'css-loader' ]   //  <=  Order is very important
            }
        ]
    },
    resolve: {
        alias: {
            'vue$': 'vue/dist/vue.esm.js',
            'numeric-keyboard$': 'numeric-keyboard/dist/numeric_keyboard.vue.js',
            'vue-loading-overlay': 'vue-loading-overlay/dist/vue-loading.min.js',
            'jquery': 'jquery/dist/jquery.js'
        }
    },
    devServer: {
        historyApiFallback: true,
        noInfo: true
    },
    performance: {
        hints: false
    },
    devtool: '#eval-source-map'
}

if (process.env.NODE_ENV === 'production') {
    module.exports.devtool = '#source-map'
    // http://vue-loader.vuejs.org/en/workflow/production.html
    module.exports.plugins = (module.exports.plugins || []).concat([
        new webpack.DefinePlugin({
            'process.env': {
                NODE_ENV: '"production"'
            }
        }),
        new webpack.optimize.UglifyJsPlugin({
            sourceMap: true,
            compress: {
                warnings: false
            }
        }),
        new webpack.LoaderOptionsPlugin({
            minimize: true
        })
    ])
}
```

**Comment by kernelcoredump** [comment]
@mazurva What's in your `package.json`? Check to make sure that `vue` is in your list of dependencies.  vue-loading-overlay makes the assumption that the consuming app/package will also reference vue.

**Comment by mazurva** [comment]
package.json
```
{
  "name": "casher-project",
  "version": "0.0.1",
  "author": "Vadim Mazur <mazurva@gmail.com>",
  "private": true,
  "scripts": {
    "dev": "cross-env NODE_ENV=development webpack-dev-server --open --hot",
    "build": "cross-env NODE_ENV=production webpack --progress --hide-modules"
  },
  "dependencies": {
    "axios": "^0.16.2",
    "jquery": "^3.3.1",
    "numeric-keyboard": "0.0.5",
    "vue": "^2.5.16",
    "vue-loading-overlay": "^2.0.2",
    "vue-native-websocket": "^2.0.5",
    "vue-router": "^2.7.0",
    "vuex": "^3.0.1"
  },
  "devDependencies": {
    "babel-core": "^6.0.0",
    "babel-loader": "^6.0.0",
    "babel-preset-env": "^1.5.1",
    "cross-env": "^3.0.0",
    "css-loader": "^0.25.0",
    "file-loader": "^0.9.0",
    "less": "^2.7.3",
    "less-loader": "^4.1.0",
    "node-sass": "^4.8.2",
    "sass-loader": "^6.0.7",
    "style-loader": "^0.20.3",
    "vue-loader": "^12.1.0",
    "vue-template-compiler": "^2.5.16",
    "webpack": "^2.6.1",
    "webpack-dev-server": "^2.11.2"
  }
}
```

**Comment by ankurk91** [comment]
@mazurva
Thanks for reporting.
This issue has been fixed in v2.0.3.
